i18n-scripts/set-english-defaults.js

```diff
@@ -0,0 +1,68 @@
+const fs = require('fs');
+const path = require('path');
+const pluralize = require('pluralize');
+const common = require('./common.js');
+
+const publicDir = path.join(__dirname, './../locales/');
+
```

Comment:
@yaacov Wondering if the path should be './locales' since the locales directory is in the root of the project.

Reply:
AFIU the path is relative to this file, this file is in `./i18n-scripts` and you need to go back one step `./..` to get the project root.
I run this script using `yarn`, and I thought it worked :-), am i missing something?

Reply:
@cyril-ui-developer hi, I removed the copied OWNER file in a seperate commit, please re-review